src/components/HomePage/SearchResultsArea/SearchResultsWidget/SearchResultsUtils.js

```diff
@@ -0,0 +1,4 @@
+export const SearchResultsConstants = {
+    initialLimit: 5,
+    fetchNewOffersLimit: 5,
```

Comment:
Make uppercase, also I'd say initial at least enough to fill the vertical space (15?) And then new offer batches of 10